src/core/location/location.controller.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  Controller,
+  Get,
+  NotFoundException,
+  Param,
+  Query,
+} from '@nestjs/common';
+import { LocationService } from './service/location.service';
+import { ApiOkResponse, ApiQuery, ApiTags } from '@nestjs/swagger';
+import { ApiInternalServerError } from 'src/common/decorator/internal-server-error-decorator';
+import { ApiException } from '@nanogiants/nestjs-swagger-api-exception-decorator';
+import { LocationResponse } from './dto/response.dto';
```

Comment:
Basta con Location, no con LocationResponse XD, pero x
```suggestion
import { Location } from './dto/response.dto';
```